### PR TITLE
CY-1717 Get the private_ip from the reporter config

### PR DESCRIPTION
**Pull request by inbala**
Also, remove get_cloudify_config function.

### CODE REVIEW

File: status-reporter/status_reporter/postgresql_reporter.py [hunk 1]
```diff
@@ -45,11 +45,12 @@ def _collect_status(self):
         return status, services
 
     def _get_patroni_status(self):
-        config = get_cloudify_config(logger)
-        if not config:
+        extra_config = self._config.get('extra_config', {})
```
Comment (tyacbovi) [inline status-reporter/status_reporter/postgresql_reporter.py]: There is not const for 'extra_config'?

File: status-reporter/status_reporter/status_reporter.py [hunk 2]
```diff
@@ -152,8 +151,8 @@ def _collect_status(self):
 
     def _report(self):
         status, services = self._collect_status()
-        if not isinstance(status, dict):
-            logger.error('Ignoring status: expected a report in dict format,'
+        if not isinstance(services, dict):
```
Comment (tyacbovi) [inline status-reporter/status_reporter/status_reporter.py]: in Yoni's PR there is the range validation for status?

Comment (inbala) [reply status-reporter/status_reporter/status_reporter.py]: Yes Yoni will update it in his PR

File: status-reporter/status_reporter/postgresql_reporter.py (hunk 1 above)
Comment (inbala) [reply status-reporter/status_reporter/postgresql_reporter.py]: There is now :)
There was in the manager-install repo

**Pull request merged**